src/astro.js

```diff
@@ -4,28 +4,44 @@ import {
   telephotoCamera,
   calibrateGotoCmd,
   startGotoCmd,
+  stopGotoCmd,
   binning2x2,
   fileTiff,
   takeAstroPhotoCmd,
   takeAstroDarkFramesCmd,
   darkGainDefault,
   utcURL,
+  timeZoneURL,
   stopAstroPhotoCmd,
   rawPreviewContinousSuperimpose,
   queryShotFieldCmd,
+  astroAutofocusCmd,
   setRAWPreviewCmd,
 } from "./api_codes.js";
-import { nowUTC, nowLocal, nowLocalFileName } from "./api_utils.js";
+
+import { nowUTC, nowLocalFileName } from "./api_utils.js";
 
 /**
- * 4.1.1 UTC+0 time
+ * 4.1.0 UTC+0 time
  * @param {string} IP
  * @returns {string}
  */
 export function formatUtcUrl(IP) {
   return `${utcURL(IP)}${nowUTC()}`;
 }
 
+/**
+ * 4.1.1 TimeZone name
+ * @param {string} IP
+ * @param {string} timezone
+ * @returns {string}
+ */
+export function formatTimeZoneUrl(IP, timezone) {
+  //return `${timeZoneURL(IP)}Europe/Paris}`;
+  // can't get it working !
+  return `${timeZoneURL(IP)}${timezone}`;
+}
+
```

Comment:
If this code doesn't work, then don't merge it into main.

Reply:
Sorry it's ok , now forget to remove the comment ..

Reply:
Made this correction on 0.0.2-beta.4